packages/mui-docs/src/branding/brandingTheme.ts

```diff
@@ -40,17 +38,15 @@ declare module '@mui/material/styles/createPalette' {
   interface TypeText {
     tertiary: string;
   }
-}
 
-declare module '@mui/material/styles/createTypography' {
-  interface TypographyOptions {
+  interface TypographyVariantsOptions {
```

Comment:
What is `TypographyVariantsOptions` and `TypographyVariants` used for? I thought it was to override the existing `TypographyOptions` and `Typography` types, but changing the name removes the override. What am I missing?

Reply:
the interface is renamed when exported from /styles. this export is the only one we support, so that's the name we must override